testing/E2E_Test_Update.md

```diff
@@ -0,0 +1,163 @@
+# Update E2E Test Proposal
+
+## Document Purpose
+
+The purpose document and the supporting documents that are created along with it 
+are to: 
+
+* outline possible approaches to writing tests and documentation on how to implement 
+ them
+* provide an update on current state of e2e testing
+* provide documentation on Appium and tools for help in creation of e2e tests
+* provide an estimation on the time required for developing e2e tests
+
+## Supporting Documentation
+
+[Previous Test Proposal](./supporting_docs/previous_test_proposal.md)
+
+[Appium Background Information](./supporting_docs/appium_information.md) - Contains 
+information on Appium, its clients and some important considerations when selecting 
+an Appium client library. 
+
+[UI Selectors And Their Importance](./supporting_docs/ui_selectors_and_their_importance.md) - 
+Contains information on UI selectors, thier impact on a UI test suite, and examples 
+of how to implement some prefered selectors in mobile applications.  
+
+[Tools for Finding UI Selectors](./supporting_docs/tools_to_find_selectors.md) - 
+Contains information on tools that can be used to identify selectors and generate
+unrefined e2e test code.
+
+## The Previous Test Proposal
+
+The previous test prosal sections on test suites and test infrastructure can be
+found at the following links:
+
+[Proposal on Test Suite](./e2e_drafts/)
+
+[Proposal on Test Infrastructure](./e2e_drafts/E2E_Test_infrastructure.md)
+
+### Test Suite Proposal Summary
+The main criteria that were used to create the proposal for the test suite 
+for end to end UI tests, were:
+
+* Tests must be readable to enable incoming developers to get involved 
+quickly in the maintenance of the test suite and production of new tests.
+
+* The test suite should be easily maintainable to reduce time spent on 
+fixing bugs and the amount of developers required to maintain it.
+
+* In order to fulfill the required test cases, the framework used to create 
+the test suite needs to be able to handle interactions from applications 
+than redirects it to the browser for signing into the application.
+
+During the research phase, various test frameworks for android apps were 
+investigated including UIAutomator, Espresso and Appium multiple appium clients 
```

Comment:
Perhaps change 'Appium multiple appium clients' to 'Appium with multiple appium clients'